cocos/scripting/js-bindings/manual/jsb_websocket.cpp

```diff
@@ -271,7 +271,9 @@ static bool WebSocket_finalize(se::State& s)
         CCLOGINFO("WebSocket (%p) isn't closed, try to close it!", cobj);
         cobj->closeAsync();
     }
-
+    if(se::ScriptEngine::getInstance()->isInCleanup()) {
+        return true;
+    }
```

Comment:
Why add it? Will it cause memory leak?

Reply:
We found that there is an issue with the double free pointer when restarting in jsvm, and it can only be returned here. This modification does not affect other scenarios

Reply:
But release is not invoked. I don't think it is a good way to resolve the issue for jsvm.

Reply:
fix，I made some modifications and called release

Reply:
The logic is different from original. You should find the reason why it is double released.